requirements/test.txt

```diff
@@ -19,3 +19,6 @@ nltk>=3.6
 
 # add extra requirements
 -r image.txt
+
+mir_eval>=0.6
+https://github.com/aliutkus/speechmetrics/archive/refs/heads/master.zip
```

Comment:
would be better to have them as PyPI package for simple caching - https://github.com/aliutkus/speechmetrics/issues/26